### PR TITLE
require the open-source pyang so as to make coverage work

### PR DESCRIPTION
**Pull request by martin-volf**
Removing the path pointing to the NSO distribution of pyang and adding pyang to pypi requirements; plus few minor related changes.

### CODE REVIEW

**Review by jojohans — APPROVED** [review]
Let me know when you merge this, I have to remember to update my containers.

**Review by micnovak — APPROVED** [review]

**Comment by martin-volf** [comment]
Fixed so that XMNR runs fine without pyang as long as coverage testing is not required.

File: drned/drned/__init__.py [hunk 1]
```diff
@@ -1,10 +1 @@
-"""@package drned
-Documentation for this module.
-
-More details.
-"""
 from .device import Device
-from .node import Node
```
Comment (jomiklos) [inline drned/drned/__init__.py:7]: this leads to implicit reexports that lead to problems with type annotations further down the line...
e.g. schema is being still imported from previous file despite explicit export being removed here...

Comment (martin-volf) [reply drned/drned/__init__.py:7]: No, `schema` is not imported in `device`; and I'm failing to see any (potential or actual) problems with type annotations, the module or its types are not used except in `test_coverage`.

**Review by jomiklos — APPROVED** [review]

**Pull request merged**